README.md

```diff
@@ -5,32 +5,32 @@ Simon Hughes
 
 20170329
 
-Version 1.00
+Version 0.1.0
```

Comment:
Can it be anymore 0.1.0? We lready at 1.0, so it should be 1.1.0 or 2.0.0 or similar.